daemon/api_quotas_test.go

```diff
@@ -59,14 +59,17 @@ func (s *apiQuotaSuite) SetUpTest(c *check.C) {
 
 	// POST requires root
 	s.expectedWriteAccess = daemon.RootAccess{}
+
+	d.Overlord().Loop()
```

Comment:
that might avoid to do this and also allow to check that EnsureBefore was called

Reply:
done